src/View/Home/index.html.twig

```diff
@@ -1,5 +1,37 @@
 {% extends 'layout.html.twig' %}
 
+    {% block title %}
+        Home
+    {% endblock %}
+
+    {% block link %}
+        <link rel="stylesheet" href="/assets/css/home.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Passero+One&display=swap" rel="stylesheet">
```

Comment:
It'll probably be better to serve you fonts locally and not rely on some external web server